src/templates/XmlRenderer/types_file.tsx

```diff
@@ -0,0 +1,5 @@
+import { Document } from "@govtechsg/decentralized-renderer-react-components";
+
```

Comment:
filename can follow camelcase (same as the other templates) instead of underscore?

Reply:
Done.